### DISCUSSION TITLE
How to atomatically log install event

### DISCUSSION THREAD

**Discussion by vytautas-pranskunas-** [oddbit/flutter_facebook_app_events#198] [Q&A]
Hello,

I see https://developers.facebook.com/docs/app-events/getting-started-app-events-android#7--add-app-events that events like install are automatically logged once SDK is installed. However i do not see any example how is it working here?
Is it enought to execute ```static final facebookAppEvents = FacebookAppEvents();``` in main.dart or should I have to do anything else?
If app is installed from app store how facebook SDK knows that app store was visited from and add?

Should I add this:
```
<meta-data
    android:name="com.facebook.sdk.CodelessDebugLogEnabled"
    android:value="true" />
```
and
```
<key>FacebookCodelessDebugLogEnabled</key>
<true/>
```

**Comment by mdusaama0** [comment]
hey did you get solution?

**Comment by chiangandy** [comment]
I have same question and confuse me. After I install facebook_app_event package plugin asnd just want to check lunch app log. However, I cannot see any log information in log and event see any request in network in flutter devtools. So confuse me, how do I know the event log is sent to facebook hub?
